scripts/deploy_manage_ec2.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+npm install -g serverless
```

Comment:
should this be moved to .travis.yml and replaced with checking serverless is installed? You may want to run this from a workstation, and it'd be a bit of a surprise if it installed a global npm package.